libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -40,10 +40,16 @@ namespace
 template <class T, class B>
 bool hasEqualNameAndParameters(T const& _a, B const& _b)
 {
+	auto makeFunctionType = [](auto const& _decl) {
+		if constexpr (std::is_same_v<std::decay_t<decltype(_decl)>, FunctionDefinition>)
```

Comment:
Fun

Reply:
It's temporary :-).

Reply:
As in gone again in a subsequent commit.